_data/gap_releases.json

```diff
@@ -0,0 +1,137 @@
+    "checksum": "cc76ecbe33d6719450a593e613fb87e9e4247faa876f632dd0f97c398f92265d"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": true,
+    "name": "v4.13.0-alpha2",
+    "publishedAt": "2024-02-19 13:08:04+00:00",
+    "tagName": "v4.13.0-alpha2",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.13.0-alpha2/gap-4.13.0-alpha2.tar.gz",
+    "checksum": "5e29ae165ebc5b81a4f038a4f71105dca86e58f9973d6efdf24e1782005d5289"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": true,
+    "name": "v4.13.0-alpha1",
+    "publishedAt": "2024-02-05 09:08:32+00:00",
+    "tagName": "v4.13.0-alpha1",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.13.0-alpha1/gap-4.13.0-alpha1.tar.gz",
+    "checksum": "4f944d5c3d0b709fdd5b1e3c72c586bfad922089a6371f2fe814c4f1d41cce79"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "v4.12.2",
+    "publishedAt": "2022-12-18 02:36:56+00:00",
+    "tagName": "v4.12.2",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.12.2/gap-4.12.2.tar.gz",
+    "checksum": "672308745eb78a222494ee8dd6786edd5bc331456fcc6456ac064bdb28d587a8"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "v4.12.1",
+    "publishedAt": "2022-10-20 20:26:03+00:00",
+    "tagName": "v4.12.1",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.12.1/gap-4.12.1.tar.gz",
+    "checksum": "f9ebef11ee31b210ce36e3c70960742b4e253282bbd5270adc9324273c92b016"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "v4.12.0",
+    "publishedAt": "2022-08-18 21:28:50+00:00",
+    "tagName": "v4.12.0",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.12.0/gap-4.12.0.tar.gz",
+    "checksum": "6005d0c0316fe8803d84681f30be64eace83a9542f975f5cb3dedfadd4391849"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "v4.11.1",
+    "publishedAt": "2021-03-02 16:33:11+00:00",
+    "tagName": "v4.11.1",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.11.1/gap-4.11.1.tar.gz",
+    "checksum": "6635c5da7d82755f8339486b9cac33766f58712f297e8234fba40818902ea304"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "",
+    "publishedAt": "2020-03-06 11:08:18+00:00",
+    "tagName": "v4.11.0",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.11.0/gap-4.11.0.tar.gz",
+    "checksum": "6fda7af23394708aeb3b4bca8885f5fdcb7c3ae4419639dfb2d9f67d3f590abb"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "v4.10.2",
+    "publishedAt": "2019-06-27 11:55:15+00:00",
+    "tagName": "v4.10.2",
+    "url": "https://github.com/gap-system/gap/releases/download/v4.10.2/gap-4.10.2.tar.gz",
+    "checksum": "11175bed0234101643d510d4ab94c44db30bd303bfe63d86b9b86ae67cff9e49"
+  },
+  {
+    "isLatest": false,
+    "isPrerelease": false,
+    "name": "",
```

Comment:
I would suggest we drop the `name` fields (they are "undefined" semantically).

Perhaps instead we could have a `version` field. Or perhaps we just say this is implied by the `tagName`...